src/linpred.jl

```diff
@@ -113,14 +128,44 @@ end
 Base.cholfact{T}(p::SparsePredChol{T}) = copy(p.chol)
 Base.cholfact!{T}(p::SparsePredChol{T}) = p.chol
 
+"""
+    coef(x)
+
+Extract the estimates of the coefficients in the model.
+
+#### Arguments:
+* `x`: Inputs can either `LinPred` or `LinPredModel`.
+
+#### Examples:
+Refer example for `glm()`
+```
+julia> coef(OLS)
+2-element Array{Float64,1}:
+ -0.666667
+  2.5 
+```
+The output is an array, where the first element is the intercept 
+and the second element is the coefficient of the `x` component.
+"""
+function coef end
+
 coef(x::LinPred) = x.beta0
 coef(x::LinPredModel) = coef(x.pp)
 
+"""
+Degrees of freedom for residuals, when meaningful.
```

Comment:
No, these are the "residual degrees of freedom". "When meaningful" should be removed (could be added to any Julia function).